core/tests/protractor_desktop/publicationAndLibrary.js

```diff
@@ -217,14 +217,23 @@ describe('Permissions for private explorations', function() {
     explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
     explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
   });
-  it('should be correct for collaborators', function() {
+  it('should not allow adding users if title is not given', function() {
     users.createUser('alice@privileges.com', 'alicePrivileges');
     users.createUser('bob@privileges.com', 'bobPrivileges');
```

Comment:
Do we need to create this many accounts here?

Reply:
We create accounts and then navigate to settings tab and because this was the first test, I felt we can create the accounts here itself.
Also we are trying to add users and verifying that we can not unless a title exists, so creating users under this test should not be a problem in my opinion.

Reply:
This isn't a problem, but I don't think it's good to add code/steps in a block which isn't going to get used in that specific block. (I think these accounts are used in the next `it` test, so that would be a better place for this step.)